app/controllers/api/v2/sessions_controller.rb

```diff
@@ -0,0 +1,11 @@
+module API
+  module V2
+    class SessionsController < ApplicationController
```

Comment:
Currently flawed concept 

Endpoint needs to be a different auth mechanism

Cannot use `v2` auth as this end point is the precursor to being able to use `v2`

May need a `v3` auth to established client to client trusted communication vector